src/lsptoolshost/onAutoInsert.ts

```diff
@@ -40,7 +40,15 @@ export function registerOnAutoInsert(languageServer: RoslynLanguageServer) {
 
             source.cancel();
             source = new vscode.CancellationTokenSource();
-            await applyAutoInsertEdit(e, changeTrimmed, languageServer, source.token);
+            try {
+                await applyAutoInsertEdit(e, changeTrimmed, languageServer, source.token);
```

Comment:
So what (if anything) is actually going to cancel this token? Because if nothing, then how do we get that CancellationError instead?

Reply:
The vscode jsonrpc can return a cancellation error if the server shuts down for example.  So it doesn't have to be this token cancelling.